### ISSUE TITLE
Twemoji not working properly when offline

### ISSUE THREAD

**Issue by desk7** [FlorianWoelki/obsidian-icon-folder#143]
**Describe the bug**
When Obsidian can't access Internet, Twemojis are not shown properly. Both in the icon selector and in the obsidian explorer, a generic thumnail and a similar Twemoji icon are shown.

**To Reproduce**
1. Put your PC offline
2. Try to add icon to a folder and search for a Twemoji icon

**Expected behavior**
Twemoji should be shown like other icons. Maybe Twemoji icon should be available offline.

**Screenshots**
<img width="532" alt="obsidian-icon" src="https://user-images.githubusercontent.com/46022640/210449576-fa17fcb6-8ef2-4acf-9ceb-225f5c6acc9e.png">

**Additional context**
If Obsidian can access internet, there is no problem.
If I add icons to folder when internet connection is enabled, then the Twemoji icons in the obsidian explorer are shown properly even when offline.

- Obsidian: 1.1.9
- obsidian-icon-folder: 1.5.1

**Comment by FlorianWoelki** [comment]
Thanks for reporting this issue. I am aware of this. 
Twemoji is currently integrated into the plugin directly without any way of download this pack or something else. I'll try to search for a solution to download the Twemoji pack, the same like the icon packs.

**Comment by pyxelr** [comment]
@FlorianWoelki: On my side, today I have noticed such a `Change icon` menu:

![2023-01-10_15-54-44](https://user-images.githubusercontent.com/30161753/211584467-7e06e835-d57f-44be-85af-34f43470b25e.png)

In general, the previously added icons are still displayed, whereas the newly defined ones are producing simply a blank space (as next to `6_Sport`):

![2023-01-10_15-56-23](https://user-images.githubusercontent.com/30161753/211584804-3e1492cf-5bee-4545-b66c-ef2c5dae90c1.png)

If I wait for a while, I will get 2 icons, like the ones posted by the author of this issue: image placeholder icon + twemoji. I've tried restarting Obsidian, as well as using macOS and Windows but the same issue occurs on both systems.

**Comment by dljsjr** [comment]
@pyxelr That might be related to https://github.com/FlorianWoelki/obsidian-icon-folder/pull/145

**Comment by FlorianWoelki** [comment]
@pyxelr should be fixed now in [1.6.0](https://github.com/FlorianWoelki/obsidian-icon-folder/releases/tag/1.6.0). Thanks to @dljsjr for the quick response.

**Comment by pyxelr** [comment]
Thanks a lot guys!

@FlorianWoelki: I can confirm that it is fixed now. Although, after updating the plugin, the `Emoji Style` field had been selected to `None`, so I had to modify it either to `Native` or `Twemoji` (which I prefer).

**Comment by FlorianWoelki** [comment]
> Thanks a lot guys!
> 
> 
> 
> @FlorianWoelki: I can confirm that it is fixed now. Although, after updating the plugin, the `Emoji Style` field had been selected to `None`, so I had to modify it either to `Native` or `Twemoji` (which I prefer).

Yes the default for the Emoji style is None. I'll write that in the release.
